src/bridge/ex_data_util/engine_data.h

```diff
@@ -39,10 +39,11 @@ namespace bridge {
 class EngineData {
  public:
   EngineData(std::unique_ptr<backing::Client> client,
-             OpenSslRsaMethod rsa_method)
+             OpenSslRsaMethod rsa_method,
+             OpenSslEcKeyMethod ec_key_method)
       : client_(std::move(client)),
-        rsa_method_(std::move(rsa_method)) {}
-  ~EngineData() = default;
```

Comment:
Is deleting this line `~EngineData() = default` intentional?

Also do you want to make it virtual?

Reply:
I deleted this for style consistency: it looks like in the other class implementations in the codebase the default destructor isn't defined unless it's a virtual destructor. This doesn't immediately need to be a virtual destructor since nothing will inherit the `EngineData` class.